src/chat/services/resource-content.service.spec.ts

```
@@ -40,14 +35,12 @@ describe('ResourceContentService', () => {
   it('uses new-format chunk pipeline when resources+chunks exist', async () => {
-    const mcpClientService = {
-      callTool: jest.fn<CallTool>(async () => ({
-        raw: {},
-        texts: ['chunk body'],
-        resourceLinks: [],
-        embeddedResources: [],
-        filteredResources: [],
-      })),
+    const retrievalService = {
+      getContentsByPaths: jest.fn(
+        async (_paths: string[]) => [
```

ESLint

[error] Delete `⏎········`

(prettier/prettier)